resources/dicts/patrols/new_cat_welcoming.json

```diff
@@ -992,7 +992,7 @@
                 "weight": 20,
                 "outsider_rep": 0,
                 "new_cat": [
-                    ["meeting", "loner"]
+                    ["status:medicine cat","meeting", "loner"]
```

Comment:
This line will not change the status of the cat. A cat outside the clan will get the status "loner", even with that added tag. I tested it out and looked into the save of the cat. To assure that this cat get the "medicine cat" after deciding to join the clan we would have to look at the background and assigning a status according to that.

Reply:
do i assign one of the med cat backstories?

Reply:
In my test run it assigned the "wandering_healer2" backstory.